templates/invitation.html

```diff
@@ -1,5 +1,15 @@
-<h2>{{name}}</h2>
+<p>Hello {{name}}</p>,
 
-<p>You have an invitation to join {{establishment}}</p>
+<p>You have an invitation you to join {{establishment}}.</p>
```

Comment:
This line doesn't make any sense. There's an extra `you` in there.

Reply:
yep, I cut some content out, _badly_